apps/system/utils/kdbg_date.c

```diff
@@ -247,6 +312,20 @@ int kdbg_date(int argc, char **args)
 		printf("                'month', 'day', 'hour':'minute':'second' 'year'\n");
 		printf("                Example: Apr 21 10:35:22 1991\n");
 		ret = ERROR;
+#else
+		printf("\nUsage: date\n");
+		printf("   or: date [-s FORMAT]\n");
+		printf("Display, or Set system time and date information\n");
+		printf("\nOptions:\n");
+		printf(" -s FORMAT     Set system time in the given FORMAT\n");
+		printf("               FORMAT: MMM DD HH:MM:SS YYYY DWR DYR D\n");
+		printf("                day of week range = [Sunday = Sun, Monday = Mon, and so on...]\n");
```

Comment:
How about change first character on each word to match easily with above FORMAT?
```
Day of Week Rage (DWR) = xxxxx
```